src/bin/release-please.ts

```diff
@@ -820,6 +820,22 @@ export const parser = yargs
       setLogger(new CheckpointLogger(true));
     }
   })
+  .option('plugin', {
+    describe: 'load plugin named release-please-<plugin-name>',
+    type: 'array',
+    default: [],
+  })
+  .middleware(argv => {
+    for (const pluginName of argv.plugin) {
+      console.log(`requiring plugin: ${pluginName}`);
+      const plugin = require(pluginName.toString());
```

Comment:
You might want to do this in a `try`/`catch` and provide a nice error message.

Reply:
Done.